README.md

```diff
@@ -72,16 +81,16 @@ Please refer to [native readme](native/README.md) if want to develop native appl
 
 ## Example Project
 
-- [Example Cases](https://github.com/cocos/cocos-example-projects): Simple yet expressive demo scenes for baseline testing and topic-specific case study.
 - [Mind Your Step 3D](https://github.com/cocos/cocos-tutorial-mind-your-step): Beginner's step-by-step tutorial project repo.
-- [UI Demo](https://github.com/cocos/cocos-example-ui): Use cases for various kinds of UI components.
 - [Test Cases](https://github.com/cocos/cocos-test-projects): Unit test scenes for every engine module.
+- [Example Cases](https://github.com/cocos/cocos-example-projects): Simple yet expressive demo scenes for baseline testing and topic-specific case study.
+- [Awesome Cocos](https://github.com/cocos/awesome-cocos): You can find out other useful tools and show cases here.
 
 ## Links
 
-- [Official site](https://www.cocos.com/products#CocosCreator)
-- [Download](https://www.cocos.com/creator)
+- [Official site](https://www.cocos.com/)
+- [Download](https://www.cocos.com/creator/download)
```

Comment:
For historical reasons, the current English website is not this link. It's https://www.cocos.com/en/creator/download in my browser.